tests/testdata/stdlib/Texte/Texte.ddp

```diff
@@ -3,6 +3,48 @@ Binde "Duden/Ausgabe" ein.
 
 Schreibe (einen leeren Text) auf eine Zeile.
 
+[Entferne_Anzahl_Vorne_Mutierend]
+Der Text Entferne_Anzahl_Vorne_Mutierend_Text ist "Hello world!".
+Entferne 0 Buchstaben am Anfang von Entferne_Anzahl_Vorne_Mutierend_Text.
+Schreibe (Entferne_Anzahl_Vorne_Mutierend_Text) auf eine Zeile. [Hello World!]
+Entferne 5 Buchstaben am Anfang von Entferne_Anzahl_Vorne_Mutierend_Text.
+Schreibe (Entferne_Anzahl_Vorne_Mutierend_Text) auf eine Zeile. [ World!]
+Entferne 2 Buchstaben am Anfang von Entferne_Anzahl_Vorne_Mutierend_Text.
+Schreibe (Entferne_Anzahl_Vorne_Mutierend_Text) auf eine Zeile. [orld!]
+Entferne -21489159 Buchstaben am Anfang von Entferne_Anzahl_Vorne_Mutierend_Text.
+Schreibe (Entferne_Anzahl_Vorne_Mutierend_Text) auf eine Zeile. [orld!]
+Entferne 21489159 Buchstaben am Anfang von Entferne_Anzahl_Vorne_Mutierend_Text.
+Schreibe (Entferne_Anzahl_Vorne_Mutierend_Text) auf eine Zeile. []
+
+[Entferne_Anzahl_Vorne]
+Der Text Entferne_Anzahl_Vorne_Text ist "Hello World!".
+Schreibe Entferne_Anzahl_Vorne_Text mit den ersten 4 Buchstaben entfernt auf eine Zeile. [o World!]
```

Comment:
Sollte `Schreibe (Entferne_Anzahl_Vorne_Text mit den ersten 4 Buchstaben entfernt) auf eine Zeile. [o World!]` sein.

In den anderen Aufrufen genau so.